src/renderer/components/test-utils/get-application-builder.tsx

```diff
@@ -169,26 +188,23 @@ export const getApplicationBuilder = () => {
   );
 
   rendererDi.override(rendererExtensionsInjectable, () =>
-    computed(() => extensionsState),
+    computed(() => [...rendererExtensionsState]),
   );
 
   mainDi.override(mainExtensionsInjectable, () =>
-    computed(() => []),
+    computed(() => [...mainExtensionsState]),
   );
 
   const iconPaths = mainDi.inject(trayIconPathsInjectable);
 
-  let trayMenuItemsStateFake: TrayMenuItem[];
   let trayMenuIconPath: string;
 
   mainDi.override(electronTrayInjectable, () => ({
     start: () => {
       trayMenuIconPath = iconPaths.normal;
     },
     stop: () => {},
-    setMenuItems: (items) => {
-      trayMenuItemsStateFake = items;
-    },
```

Comment:
why remove this? Don't we want to test non-extension menu items?

Reply:
Of course we do. They can be tested just like any item originating from extension. Having this fake here, skipped some business logic we had for tray menu items. (e.g. filtering based on visibility) which is very important to be part of unit test. Changes to `applicationBuilder.tray.click` and `applicationBuilder.tray.get` made this redundant.

Reply:
I still don't understand, wouldn't it be a better to test the runables are set up correctly as well? And thus this would still be needed?

Reply:
As discussed in the huddle, this is not currently possible because of some other things. We will get to that point.